scss/components/_page-headers.scss

```diff
@@ -28,7 +28,9 @@
   border-bottom: none;
   font-family: $sans-serif;
   font-size: 1.6rem;
+  text-transform: uppercase;
```

Comment:
Properties should be ordered border-bottom, font-family, font-size, line-height, margin-bottom, margin-left, text-transform